src/main/java/dk/cngroup/lentils/controller/ProgressController.java

```diff
@@ -7,27 +7,30 @@
 import dk.cngroup.lentils.factory.CypherStatusFactory;
 import dk.cngroup.lentils.service.CypherService;
 import dk.cngroup.lentils.service.HintService;
+import dk.cngroup.lentils.service.HintTakenService;
 import dk.cngroup.lentils.service.ProgressService;
 import dk.cngroup.lentils.service.StatusService;
 import dk.cngroup.lentils.service.TeamService;
-import dk.cngroup.lentils.service.HintTakenService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.List;
 
 @Controller
 @RequestMapping("/game/progress")
 public class ProgressController {
     private static final String PROGRESS_STAGE = "progress/stage";
     private static final String PROGRESS_LIST = "progress/list";
     private static final String TEAM_LIST = "progress/teamList";
+    private static final String REDIRECT_TEAM_LIST = "redirect:/game/progress/teamsList";
```

Comment:
tady by melo byt mozna lomitko na konci, ale nejsem si jisty, je potreba otestovat ;) pamatuju si ze to nekdy delalo problemy

Reply:
noo řekl bych že tady to není potřeba, protože se za to nedává žádné IDčko, jako třeba u @GetMapping(value = "/viewHints/{cypherId}")    (nebo takto to aspoň chápu já :D )